Tests/SonarQube.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -401,9 +568,8 @@ private string CreateAndBuildSonarProject(ProjectDescriptor descriptor, string r
 
             logger.AssertExpectedTargetOrdering(
                 TargetConstants.CategoriseProjectTarget,
-                TargetConstants.CoreCompileTarget,
-                TargetConstants.CalculateFilesToAnalyzeTarget,
```

Comment:
I've fixed up the expected ordering in the existing tests:
* all references to CoreCompileTarget have been removed (although I didn't delete the CoreCompile test constant - I'll do that).

* CalculateFilesToAnalyze is only used by the "Write project info" target and should be executed just in time i.e. just before it